scripts/generate

```diff
@@ -95,6 +95,14 @@ generator=(
   --assets="configs/assets"
   --retry="${botocore}/_retry.json"
 )
+say "MODEL"
+say "${models[*]}"
+if [[ " ${models[*]} " =~ " sts " ]]; then
```

Comment:
Does this mean that if someone tries to regenerate only STS, it will update the copy inside amazonka instead?

Reply:
In this implementation `sts` updates both embedded STS in amazonka and real `amazonka-sts`.
I've thought about creating extra `sts-embedded` model, but that would be awkward and lead to config copy-paste.

Reply:
Could you please add a comment to this section?

Reply:
Sure, done.
Thank you for a prompt review!